lib/note-toolbar/index.tsx

```diff
@@ -184,4 +188,8 @@ const mapStateToProps: S.MapState<StateProps> = ({ ui: { note } }) => ({
   note,
 });
 
-export default connect(mapStateToProps)(NoteToolbar);
+const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
+  onToggleNoteInfo: toggleNoteInfo,
```

Comment:
Since we're creating the interface and the names let's abandon `on_` nomenclature - stick to the active voice and indicate what we want, not _when_ a thing happens.

```ts
const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
  toggleNoteInfo
}
```